### ISSUE TITLE
When I zoom in to the maximum level, the curve becomes a triangle

### ISSUE THREAD

**Issue by zjinger** [elfalem/Leaflet.curve#56]
hello
When I zoom in to the maximum level and drag the map, the curve becomes a triangle
![image](https://user-images.githubusercontent.com/17251303/208817232-adff5f02-5f3e-41a8-b6dc-31a3cc5c0058.png)

**Comment by Br4v3H3r0** [comment]
Yes! I'm facing the same issue, for now I'm keeping the MAX_ZOOM_LEVEL of map at around 12 to avoid this formation of triangles until this gets fixed

**Comment by elfalem** [comment]
Can I get more details on how to reproduce this issue?

**Comment by Br4v3H3r0** [comment]
> Can I get more details on how to reproduce this issue?

As mentioned by @zjinger , on the demo website provided in README, which is https://elfalem.github.io/Leaflet.curve/,
pick one of the blue curves and zoom in to one of the ends of the curve until you reach the MAX_ZOOM level (which is I believe 18), you will notice that the curve turns into a triangle.

**Comment by elfalem** [comment]
I tried zooming in on the ends of a curve. I do not see the curve having the thickness shown above or becoming a triangle. I think I'm still missing some key information.

![image](https://user-images.githubusercontent.com/11811344/223586257-918d6cdd-58b9-4a19-9c2a-5b9bef926c31.png)

**Comment by Br4v3H3r0** [comment]
Here is a video demonstration for reproducing the issue. I'm using the Brave Browser.
[Video Link](https://drive.google.com/drive/folders/1foHdtvkxCRsSeEzUC9bZLIPWXF_RRGHT?usp=sharing)

**Comment by elfalem** [comment]
Thanks a lot for the video demonstration. I'll investigate.

**Comment by bachbone** [comment]
any progress on this issue?
